test/season_statistics_test.rb

```diff
@@ -0,0 +1,42 @@
+require "./lib/season_statistics"
```

Comment:
@Arique1104 I added the test that we wrote for the helper method games by season - I did not add your fixture files. I think you might want to look over it again and draw a sample that will work for your methods.